### DISCUSSION TITLE
J-Stars in-game bug

### DISCUSSION THREAD

**Discussion by Darksoul12344** [community/community#47508] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Im having a bug in the game J-Stars where ingame there show's no characters but game does play. Please help me with this problem

**Comment by queenofcorgis** [comment]
Hi @Darksoul12344 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on J-Stars. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
